lib/matplotlib/transforms.py

```diff
@@ -2062,6 +2062,14 @@ def frozen(self):
         # docstring inherited
         return self
 
+    def __deepcopy__(self, memo):
+        """
+        The identity transform does not need to lock out deepcopy
+        """
+        return self.frozen()
```

Comment:
I'd put this into a comment. It's not a documentation of the functionality (this message is not relevant to the user), but a comment on an implementation detail. Also we can directly return `self` to make it more explicit what we're doing anyway.

```suggestion
        # The identity transform is not immutable, so we don't need to copy.
        return self
```